mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -742,49 +742,45 @@ trait Completions { this: MetalsGlobal =>
         interpolator: InterpolationSplice,
         text: String
     ) extends CompletionPosition {
+
       val offset = if (lit.pos.focusEnd.line == pos.line) CURSOR.length else 0
-      val litpos = lit.pos.withEnd(lit.pos.end - offset)
+      val litpos = pos.withStart(pos.start - interpolator.name.size)
```

Comment:
can we use `literalPosition` or at least `literalPos` ?

Reply:
Fixed